tests/test_vcs_lambert_azimuthal.py

```diff
@@ -0,0 +1,17 @@
+import basevcstest
+
+class TestVCSLambertAzimuthal(basevcstest.VCSBaseTest):
+    def allTests(self, centerlongitude):
```

Comment:
@doutriaux1 It runs every function that starts with test, so that one is fine. But I like better main so I'll change it to that.